### ISSUE TITLE
Integrate this project with Coverity

### ISSUE THREAD

**Issue by monsdar** [monsdar/CxxProf#45]
[Coverity](https://scan.coverity.com) provides free static analysis for open source projects. It is possible to use [Travis CI](https://scan.coverity.com/projects/2653/builds/new?tab=travis_ci) as a base, coverity is then run from it as an additional build step.

I created an [account for CxxProf](https://scan.coverity.com/projects/2653?tab=overview) but haven't been able to configure everything as it needs to be.

**Comment by anirbanmu** [comment]
I'd love to work on this, should I convert this issue to your GitLab repo? @monsdar

**Comment by monsdar** [comment]
Yeah, Gitlab is where I made the latest changes - mainly integration of Conan, modularization and C++11.
I'm not working with C++ anymore, so this is why the project fell asleep. Would be happy if someone starts working on it again.
